content/spin/plugin-authoring.md

```diff
@@ -0,0 +1,41 @@
+title = "Creating Spin plugins"
+template = "spin_main"
+date = "2023-01-26T00:22:56Z"
+[extra]
+url = "https://github.com/fermyon/spin/blob/main/docs/content/plugin-authoring.md"
```

Comment:
```suggestion
```

Reply:
Referencing the unpublished page seems to be an issue. So just removing the link.

Reply:
How are we supposed to add this extra info? Were all the others like the one in templates added after the initial merge?